tests/ga/test_exthandlers.py

```diff
@@ -249,29 +249,6 @@ def test_extension_sequence_number(self):
                                               disk_sequence_number=3,
                                               expected_sequence_number=-1)
 
-    @patch("azurelinuxagent.ga.exthandlers.add_event")
-    @patch("azurelinuxagent.common.errorstate.ErrorState.is_triggered")
-    def test_it_should_report_an_error_if_the_wireserver_cannot_be_reached(self, patch_is_triggered, patch_add_event):
```

Comment:
This is a redundant test, this scenario is being tested by the tests for `try_update_goal_state` functionality